interfaces/overview/src/app/services/jwt.service.ts

```diff
@@ -0,0 +1,26 @@
+import { Injectable } from '@angular/core';
+import { JwtHelperService } from '@auth0/angular-jwt';
```

Comment:
I think we could use this package even more, so we don't need to handle the headers in the ApiService and we don't need this whole JWT-Service.
https://github.com/auth0/angular2-jwt#usage-injection

But it might be more work to refactor this then it saves at the moment...